buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy

```diff
@@ -112,6 +113,7 @@ SHORT_NAMES = ['all' : ['ppc64le_linux','ppc64le_linux_xl','s390x_linux','s390x_
             'zlinuxlargeheap' : ['s390x_linux_xl'],
             'zlinuxxl' : ['s390x_linux_xl'],
             'plinux' : ['ppc64le_linux'],
+            'plinuxcm' : ['ppc64le_linux_cm'],
```

Comment:
xlinux supports xlinuxcm and xlinuxcmake. I'm fine with only supporting the shorter version as long as there is agreement and awareness.